bashrc

```diff
@@ -2,4 +2,4 @@ cat << EOF
 Welcome to your development environment. Happy coding!
 EOF
 
-export PS1="\[\e[32m\]okteto\[\e[m\]> "
+export PS1="\[\e[32m\]\${OKTETO_NAMESPACE:-okteto} \w\[\e[m\]> "
```

Comment:
@rlamana anything else we should have here? or a different format?
This one looks like this if you are in the namespace `okteto`:

<img width="310" alt="Screenshot 2020-04-19 at 10 13 15" src="https://user-images.githubusercontent.com/7474696/79682903-74502900-8226-11ea-97f6-5f605118bb0e.png">

Reply:
The idea with the custom prompt was to make it obvious that this was the remote shell, and not the local one. That's why we went with `okteto>`

I think showing the namespace is useful, but the prompt looks odd no? maybe we need to add a separator between the namespace and the rest?, Or have the namespace on a different color?

Having such a long bash prompt takes space away from the commands I'm typing, so that's something else to consider

Reply:
I like the second option, I think the color and the  word `okteto` make it more clear.

Reply:
In that case we should show current directory and not the absolute path, to limit the size of the prompt a little bit

Reply:
<img width="231" alt="Screenshot 2020-04-28 at 16 03 01" src="https://user-images.githubusercontent.com/7474696/80496665-cf41f880-8969-11ea-8150-67a08b554240.png">

namespace:name path>

Reply:
`hello-world` is the name of the dev env?

Reply:
@rberrelleza yes